### PR TITLE
Home bugs

### PR DESCRIPTION
**Pull request by naina66406**
Please always provide the [GitHub issue(s)](../issues) your PR is for, as well as test URLs where your change can be observed (before and after):

Fix #<gh-issue-id>

Test URLs:
- Before: https://main--world-bank--aemsites.hlx.live/
- After: https://home-bugs--world-bank--aemsites.hlx.live/en/leadership/axel-van-trotsenburg

### CODE REVIEW

File: blocks/bio-detail/bio-detail.js [hunk 1]
```diff
@@ -41,6 +41,10 @@ function createPersonBio(
   resources,
   block,
 ) {
+  const pElement = document.querySelector('.display-name p');
```
Comment (mkbansal1) [inline blocks/bio-detail/bio-detail.js:44]: avoid document.querySelector, use block. instead

File: blocks/bio-detail/bio-detail.js [hunk 2]
```diff
@@ -119,12 +122,14 @@ export default async function decorate(block) {
...
   );
 
-  const resourcesTargetDiv = document.querySelector('.resources');
-  resourcesTargetDiv.insertBefore(
-    div({ class: 'title' }, placeholderData.biodetailResourcesText),
-    resourcesTargetDiv.firstChild,
-  );
-  createResources(block);
+  if (resources.textContent.trim()) {
+    const resourcesTargetDiv = document.querySelector('.resources');
```
Comment (mkbansal1) [inline blocks/bio-detail/bio-detail.js:126]: dont use document.querySelector

File: styles/lazy-styles.css [hunk 3]
```diff
@@ -156,7 +156,7 @@ body.page-swoosh-half-curve::before { background-image: url('../icons/swoosh/pag
     }
 }
 
-@media (width >= 1024px) {
+@media (width > 1024px) {
     .columns.columns-transform-teaser .columns-img-col::before {
```
Comment (mkbansal1) [inline styles/lazy-styles.css:160]: Desktop is >= 1024

File: blocks/columns/columns.css [hunk 4]
```diff
@@ -305,7 +305,7 @@
   padding-left: 10px;
 }
 
-@media(width<=1024px) {
+@media(width <= 1024px) {
   .bg-primary-blue-90 .columns.columns-transform-teaser p:nth-child(1) {
```
Comment (mkbansal1) [inline blocks/columns/columns.css:309]: this should be <1024

**Review by mkbansal1 — CHANGES_REQUESTED** [review]
@naina66406  Please check the comments inline

File: blocks/bio-detail/bio-detail.css [hunk 5]
```diff
@@ -96,8 +99,8 @@
 
 .bio-detail .sections-container .name-job-social .social-media {
   display: flex;
-  gap: 10px;
-  margin-top: 20px;
+  gap: var(--xsmall);
```
Comment (thomsebastin) [inline blocks/bio-detail/bio-detail.css:102]: `--spacing-xsmall` and `--spacing-regular` pls.

File: blocks/bio-detail/bio-detail.css [hunk 6]
```diff
@@ -198,7 +197,9 @@
   }
 
   .bio-detail .sections-container {
-    flex-direction: row;
+    /* flex-direction: row; */
+            display: grid;
+              grid-template-columns: 1fr 1fr;
```
Comment (thomsebastin) [inline blocks/bio-detail/bio-detail.css:202]: format pls

**Review by mkbansal1 — APPROVED** [review]

**Pull request merged**